.github/workflows/build_iso.yml

```diff
@@ -1,62 +1,140 @@
 name: Build Bluefin ISOs
 on:
   workflow_dispatch:
-  pull_request:
-    branches:
-      - main
-    paths:
-      - '.github/workflows/build_iso.yml'
-
-env:
-  IMAGE_REGISTRY: ghcr.io/${{ github.repository_owner }}
+  workflow_call:
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}-iso
   cancel-in-progress: true
 
 jobs:
   build-iso:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     permissions:
       contents: read
       packages: write
       id-token: write
-    container:
-      image: fedora:${{ matrix.major_version }}
-      options: "--privileged"
-      volumes:
-        - "/:/host"
     strategy:
       fail-fast: false
       matrix:
-        image_name: [bluefin, bluefin-nvidia, bluefin-asus, bluefin-asus-nvidia, bluefin-surface, bluefin-surface-nvidia, bluefin-dx, bluefin-dx-nvidia, bluefin-dx-asus, bluefin-dx-surface, bluefin-dx-asus-nvidia, bluefin-dx-surface-nvidia]
+        image_name:
+          - bluefin
+          - bluefin-nvidia
+          - bluefin-asus
+          - bluefin-asus-nvidia
+          - bluefin-surface
+          - bluefin-surface-nvidia
+          - bluefin-dx
+          - bluefin-dx-nvidia
+          - bluefin-dx-asus
+          - bluefin-dx-surface
+          - bluefin-dx-asus-nvidia
+          - bluefin-dx-surface-nvidia
         major_version: [38, 39]
-        image_tag: [latest, gts]
         exclude:
           - major_version: 38
-            image_tag: latest
+            image_name: bluefin-asus
+          - major_version: 38
+            image_name: bluefin-asus-nvidia
+          - major_version: 38
+            image_name: bluefin-dx-asus
+          - major_version: 38
+            image_name: bluefin-dx-asus-nvidia
+        include:
+          - major_version: 38
+            is_gts_version: true
           - major_version: 39
-            image_tag: gts
+            is_gts_version: false
     steps:
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@v1.3.1
```

Comment:
Should we use the ublue action?
https://github.com/ublue-os/remove-unwanted-software

Reply:
Ublue action is out of date, this one works fine.

Reply:
Out of date with upstream.  LGTM